backend/src/main/java/com/empire/Constants.java

```diff
@@ -11,6 +11,14 @@
  * the numeric data could be game config
  */
 public class Constants {
+    public static final double foodFedPointFactor = 1 / 1000000.0;
```

Comment:
I'm pretty sure this falls under my own personal style guidelines used by N=1 people but...

I prefer seeing large numbers with zeroes like theses as 1E6 etc, that way I don't need to count how many zeros there are.

Reply:
Done.